### ISSUE TITLE
Browser vendors positions on standards

### ISSUE THREAD

**Issue by MendyBerger** [mdn/browser-compat-data#8440]
This is a feature request.

I think that it would be valuable to provide the position of browser vendors on new proposals so that users know to differentiate between API's that are on the way from those that are never coming.

Perhaps there can be a system with values like: 
- In support
- Neutral
- Opposed
- No comments yet

Browser vendors share information like this: [Firefox](https://mozilla.github.io/standards-positions/), [Chromium](https://chromestatus.com/features/5659115386830848), and [Safari](https://webkit.org/status/#feature-block-scoping)

**Comment by MendyBerger** [comment]
We can map values from Firefox:
- `Important` and `Worth prototyping` = In support
- `Non harmful` and `Defer` = Neutral
- `Harmful` = Opposed
- `Under consideration` = No comments yet

**Comment by MendyBerger** [comment]
From Safari
- `In development` and `Supported in preview` = In support
- `Not considering` = Opposed
- `Under consideration` = No comments yet

**Comment by ddbeck** [comment]
(cc: @Elchi3)

Hi @MendyBerger and thank you for proposing this enhancement to BCD. I think you're right that web developers should be able to differentiate between forthcoming APIs and mere proposals, and BCD may be in a position to help with that, but vendor positions themselves are likely out-of-scope for BCD.

First, the positive news. This has significant overlap with the proposed `implementation_url` addition to the schema. See https://github.com/mdn/browser-compat-data/pull/7396 and https://github.com/mdn/browser-compat-data/pull/7396. Hopefully soon, BCD will begin experimenting with recording data for features in development, as well as those already shipping.

That said, I don't think tracking vendor positions directly is a good fit for the project, at least in the near term.

Mostly, this is a practical matter. Tracking unimplemented specifications and vendor positions is difficult because they can change radically in a short time. Waiting for some semblance of stability (such as evidence of implementation) before we start tracking a feature is a more sustainable effort for the small group of core BCD contributors.

More theoretically, vendor positions themselves don't change the compatibility story for web developers. Features which are strictly proposed and never implemented don't have a direct impact on compatibility for web developers, since they can't be used in any browser. Whether a vendor has a positive feeling toward a proposal is interesting and valuable information for developers, but it's a very expansive view of what constitutes compat data.

Again, I want to thank you for proposing this enhancement. It's been a really helpful exercise to think about your suggestion and validates some of the things I've been hoping for the `implementation_url` proposal.

**Comment by queengooborg** [comment]
New discussion has begun regarding this, see https://github.com/orgs/mdn/discussions/463.  In turn, this issue has now been reopened as there is renewed interest in implementing this!

**Comment by caugner** [comment]
The related discussion in https://github.com/orgs/mdn/discussions/463 was concluded, and a banner was added to https://developer.mozilla.org/en-US/docs/Web/API/Storage_Access_API/Related_website_sets in https://github.com/mdn/content/pull/31942.

Overall, there are 8 pages with this banner, some of which relate to the same API: https://github.com/search?q=repo%3Amdn%2Fcontent+%22This+feature+is+currently+opposed+by%22&type=code

Given that there seem to be so few affected features and that the OP received no upvotes since 2023, let's close this as wontfix.
